src/test/java/core/basesyntax/service/ReportServiceImplTest.java

```diff
@@ -0,0 +1,86 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.fail;
+
+import core.basesyntax.bd.Storage;
+import core.basesyntax.model.FruitRecordDto;
+import core.basesyntax.service.type.service.BalanceHandler;
+import core.basesyntax.service.type.service.OperationHandler;
+import core.basesyntax.service.type.service.PurchaseHandler;
+import core.basesyntax.service.type.service.ReturnHandler;
+import core.basesyntax.service.type.service.SupplyHandler;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportServiceImplTest {
+    private static ReportService reportService;
+    private static OperationStrategy operationStrategy;
+    private static final String EMPTY_PATH = "";
+    private static final String CORRECT_PATH_FROM = "src/test/resources/filesFruitShop.csv";
+    private static final String CORRECT_PATH_TO = "src/test/resources/report_fruit_shop.csv";
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        Map<FruitRecordDto.Activities, OperationHandler> mapTypeHandler = new HashMap<>();
+        mapTypeHandler.put(FruitRecordDto.Activities.BALANCE, new BalanceHandler());
+        mapTypeHandler.put(FruitRecordDto.Activities.PURCHASE, new PurchaseHandler());
+        mapTypeHandler.put(FruitRecordDto.Activities.RETURN, new ReturnHandler());
+        mapTypeHandler.put(FruitRecordDto.Activities.SUPPLY, new SupplyHandler());
+        operationStrategy = new OperationStrategyImpl(mapTypeHandler);
+        reportService = new ReportServiceImpl(operationStrategy);
+    }
+
+    @Test
+    public void getReport_correctPath_Ok() {
+        Assert.assertTrue(reportService.getReport(CORRECT_PATH_FROM, CORRECT_PATH_TO));
+    }
```

Comment:
I think we should write test, where we call `getReport` method, and than check the file content(CORRECT_PATH_TO), to ensure that our service creates right report and write it to file